src/prevector.h

```diff
@@ -220,7 +220,10 @@ class prevector {
         }
     }
 
-    prevector() : _size(0) {}
+    prevector() : _size(0) {
+        // Explicitly initialize indirect ptr to avoid "used uninitialized" warning
+        _union.indirect = nullptr;
```

Comment:
@laanwj There is a faction of language wonks/compiler folks that argues that it is never acceptable (except with char) per the language specs; -- and I believe there is nothing in any of the C++ standards that make it clear that its kosher (though sipa tells me apparently C11 has something explicit). Since having no way to do it is not a very realistic position it doesn't reflect the behavior of compilers today. Generally better to avoid it at least where its convenient, or at least be aware that someday it might become non-kosher depending on which evil spirits control the souls of the compiler authors next week. :)